casper/src/main/formal_verification/tla+/GenesisCeremony.toolbox/GenesisCeremony___Model_1.launch

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You probably want to ignore `GenesisCeremony.toolbox` folder. Are these files generated with TLA IDE?

Reply:
I put in the .gitignore to ignore .toolbox files. I'm now wondering if these files should be included though. They contain info about different model settings that are used in TLC. I'm curious what others think.

Reply:
Tomislav can be the second approver after his question is resolved. Thanks

Reply:
I'm looking into how other TLA+ projects do it.